ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -136,27 +128,34 @@ void requestBannerAd(@NonNull AdSize adSize, @NonNull MediationAdRequest mediati
     if (TextUtils.isEmpty(mPlacementId)) {
       String message = "Failed to load ad from Vungle: Missing or Invalid Placement ID.";
       Log.w(TAG, message);
-      if (bannerListener != null) {
-        bannerListener.onAdFailedToLoad(adapter, AdRequest.ERROR_CODE_INVALID_REQUEST);
-      }
-      return;
+      mVungleBannerListener.onAdFailedToLoad(AdRequest.ERROR_CODE_INVALID_REQUEST);
+      return null;
     }
 
     AdConfig.AdSize vungleAdSize = getSupportedAdSize(context, adSize);
     if (vungleAdSize == null) {
       String message = "Failed to load ad from Vungle: Invalid banner size.";
       Log.w(TAG, message);
-      if (bannerListener != null) {
-        bannerListener.onAdFailedToLoad(adapter, AdRequest.ERROR_CODE_INVALID_REQUEST);
-      }
-      return;
+      mVungleBannerListener.onAdFailedToLoad(AdRequest.ERROR_CODE_INVALID_REQUEST);
+      return null;
     }
+
+    mUniquePubRequestId = config.getRequestUniqueId();
```

Comment:
very minor: we could move that block back to https://github.com/Vungle/googleads-mobile-android-mediation/pull/67/files#diff-059c77ef8b7787eb50e9a78d86099aeaL184
and call 
'mAdLayout = new WeakReference<>(adLayout);'
after